docs/en/relying-party-solution.rst

```diff
@@ -142,16 +142,14 @@ When the flow is Cross Device, the user-agent needs to check the session status
 
 Since the QRcode page and the status endpoint are implemented by the Relying Party, it is under its responsability the implementation details of this solution, since it is related to the Relying Party's internal API.
 
-The Relyng Party MUST bind the request of the user-agent, with a Secured and Httponly session cookie, with the issued request (using the ``random-value`` parameter within the ``request_uri`` value). The HTTP response returned by this specialized endpoint MAY contain the HTTP status codes listed below:
+The Relyng Party MUST bind the request of the user-agent, with a Secured and Httponly session cookie, with the issued request. The request url SHOULD include a parameter with a random value. The HTTP response returned by this specialized endpoint MAY contain the HTTP status codes listed below:
```

Comment:
we not mandate an implementation to have a random value since the lookup parameter can be implemented on top of a cookie, we mandate the security requirements but now how these must be implemented to be satisfied. However we define a proposal with a normative language that leaves up to the implementation the technical solution